src/test/java/tc/DataProviders.java

```diff
@@ -0,0 +1,26 @@
+package tc;
+
+import org.testng.annotations.DataProvider;
+
+public class DataProviders {
+
+    @DataProvider(name = "Correct data for sum")
+    public Object[][] correctDataSum() {
+        return new Object[][] {{3, 3}};
+    }
+
+    @DataProvider(name = "Correct data for sub")
+    public Object[][] correctDataSub() {
+        return new Object[][] {{3, 3}};
+    }
+
+    @DataProvider(name = "Correct data for multiply")
+    public Object[][] correctDataMultiply() {
+        return new Object[][] {{3, 3}};
+    }
+
+    @DataProvider(name = "Correct data for div")
+    public Object[][] correctDataDiv() {
+        return new Object[][] {{3, 3}};
+    }

```

Comment:
Why do you use in all data provider same value?

Reply:
Does it matter? There were not conditions about data provider's value in the task.